### PR TITLE
fix(import_image): replace resized-image embed

### PR DESCRIPTION
**Pull request by curbengh**
fix https://github.com/hexojs/hexo-migrator-wordpress/pull/73#issuecomment-662125494 cc @adnan360

### CODE REVIEW

**Comment by curbengh** [comment]
@adnan360 should resized-images be downloaded? Currently, this PR only downloads full-sized image and replace the resize-image embed.

**Comment by curbengh** [comment]
I will add option to download resized image in next PR. @adnan360 which option should be default, original-only or original+resized?

**Comment by adnan360** [comment]
Well, I'll change my opinion a little bit. I think setting original+resized would be a better option for newbies as a default. Since, they'd just try to run it without parameters and expect it to work exactly as wp. Resized images will be converted to resized images, full-size images to full-size. No complain should be there.

But a more Hexo way would be to import them as full-size images only and install a plugin for creating resized images and painstakingly editing posts manually to use [plugin-specific code](https://github.com/ottobonn/hexo-image-sizes#embed-images) (which would be a hard thing to do for large collection of posts). Although Hexo wants to do it this way, many users may have wp sites where having the exact size from original post may be important for them.

In this whole scenario the main limitation is the replacement part. Markdown does not let us set a full size image URL to typical image code (at least I didn't find anything like that). If that was possible then we could have both original and resized and make lightbox show the full size image. But that's not possible on pure markdown+vanilla Hexo setup.

This is my personal opinion, of course. If there's an option to have full size images, I don't actually mind which is the default.

File: lib/migrator.js [hunk 1]
```diff
@@ -188,15 +192,51 @@ module.exports = async function(args) {
       }
 
       if (Object.keys(images).length && rImg.test(content)) {
-        post.content = content.replace(rImg, (matched, wpImg) => {
-          const path = images[wpImg];
+        const resizeImages = [];
+
+        // https://stackoverflow.com/a/48032528
```
Comment (curbengh) [inline lib/migrator.js:197]: :cold_sweat:

**Comment by curbengh** [comment]
@adnan360 
original+resized is now the default. for original only, `$ hexo migrate wordpress /path/export.xml --import_image original`.

**Pull request merged**